include/multiphase/private/fd_operators_inc.h

```diff
@@ -0,0 +1,514 @@
+#ifndef included_multiphase_fd_operators_inc
+#define included_multiphase_fd_operators_inc
+
+#include "multiphase/fd_operators.h"
+#include "multiphase/utility_functions.h"
+
+#include <CartesianPatchGeometry.h>
+#include <CellData.h>
+#include <SideData.h>
+
+namespace multiphase
+{
+inline void
+applyForcesOnPatchConstantCoefficient(SAMRAI::tbox::Pointer<SAMRAI::hier::Patch<NDIM>> patch,
```

Comment:
Yes. The function names should be more specific.

Reply:
Actually, looking more at this. This function _is_ for constant coefficient. Both the viscosities and drag are constant. You could consider a case in which the viscosity coefficient is variable. Further, I would argue that variable volume fractions is not "variable coefficient" because the volume fraction is a degree of freedom of the model.

I will rename them from `applyForcesX` to `accumulateMomentumForcesX`

Reply:
Ah okay. Back when we started this project we weren't including Thn as a degree of freedom and only wanted to solve stokes equations for un, us, p, hence the naming confusion. Glad to have this sorted out. I do prefer the use of the Multiphase prefix for all classes.